### ISSUE TITLE
kubernetes_ioc_activity_query: invalid SQL — INNER JOIN placed after WHERE clause

### ISSUE THREAD

**Issue by arielkr256** [panther-labs/panther-analysis#2101]
## Description

The scheduled query in `queries/kubernetes_queries/kubernetes_ioc_activity_query.yml` ("IOC Activity in K8 Control Plane") contains invalid SQL: the `INNER JOIN` clause appears *after* the `WHERE` clause.

```sql
FROM panther_logs.public.amazon_eks_audit, lateral flatten(source_ips)
WHERE p_occurs_since('30 minutes')
 -- as an example, could be replaced with any IOC data store in a lookup table
INNER JOIN panther_lookups.public.tor_exit_nodes
ON value = ip
LIMIT 10
```

Snowflake rejects it at compile time:

```
ErrorCode=001003, ErrorMessage=SQL compilation error:
syntax error line 12 at position 0 unexpected 'INNER'.
```

The query has `Enabled: false`, which is likely why this has gone unnoticed — it fails the moment anyone enables it.

## Suggested fix

Move the join before the `WHERE` clause (and relocate the inline comment):

```sql
FROM panther_logs.public.amazon_eks_audit, lateral flatten(source_ips)
-- as an example, could be replaced with any IOC data store in a lookup table
INNER JOIN panther_lookups.public.tor_exit_nodes
ON value = ip
WHERE p_occurs_since('30 minutes')
LIMIT 10
```

## How this was found

Discovered while live-testing a new `validate-sql` command for panther_analysis_tool (panther-labs/panther_analysis_tool#639), which compiles each query's SQL against the data lake via `EXPLAIN`. Static analysis (sqlfluff) had not flagged this.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

**Comment by rootverdict** [comment]
Hi @arielkr256, I’d like to submit a fix for this issue. I confirmed that develop still places the Snowflake INNER JOIN after WHERE, and I found no linked or competing PR. I’ll move the join before the filter, leave the Databricks query unchanged, and run the repository’s formatting, lint, and test workflows. I’m preparing a focused PR against develop.

**Comment by rootverdict** [comment]
Thanks @arielkr256 for reviewing and merging this! I appreciate the opportunity to contribute to Panther Analysis.
